packages/autorest.typescript/test/integration/lro.spec.ts

```diff
@@ -195,27 +195,14 @@ describe("LROs", () => {
     });
 
     it("should handle post202NoRetry204", async () => {
-      try {
-        await client.lROs.beginPost202NoRetry204AndWait(LROOptions);
-        throw new Error("should have thrown instead");
-      } catch (e) {
-        assert.equal(
-          e.message,
-          "Received unexpected HTTP status code 204 while polling. This may indicate a server issue."
-        );
-      }
+      await client.lROs.beginPost202NoRetry204AndWait(LROOptions);
+      check200(lastResponse);
```

Comment:
Why is this check200? According to the name `beginPost202NoRetry204AndWait` It should be 204

Reply:
I think it should be a 204, see https://github.com/Azure/azure-sdk-for-js/blob/1f9325c66677b85c96b3119a3a17dfe093806d31/sdk/core/core-lro/test/engine.spec.ts#L67-L96